### DISCUSSION TITLE
Dialog

### DISCUSSION THREAD

**Discussion by Rozerinay** [nl-design-system/backlog#280] [Component suggestions]
<img width="720" alt="visueel voorbeeld van een non-modal-dialog component" src="https://github.com/user-attachments/assets/5c85db9a-cf7f-447b-a928-d4813d0c1ac4">

We willen graag meer leren over de Non Modal Dialog component. In welke vorm, of vormen, wordt deze momenteel ingezet en toegepast binnen jouw organisatie?

Deel hieronder een link naar Figma, Storybook, GitHub of screenshots. Met deze informatie kunnen we bewijzen dat de component onderdeel moet worden van de NL Design System component library.

Daarnaast wordt het delen van onderzoek gewaardeerd. Dus heb je zelf onderzoek gedaan en zijn er inzichten voor dit component? Dan horen we dit ook graag!

[Link naar bericht in Slack](https://codefornl.slack.com/archives/C7KDQRXMZ/p1734523200759769)

---

## Naam
Non Modal Dialog

## Doel
Toont een venster over de pagina om de aandacht op een enkele taak of bericht te vestigen. Interactie met onderliggende pagina blijft mogelijk.

## Gerelateerde componenten
[Alert Dialog](https://github.com/orgs/nl-design-system/discussions/302), [Drawer](https://github.com/orgs/nl-design-system/discussions/238), [Modal Dialog](https://github.com/orgs/nl-design-system/discussions/237)

## Onderzoek

[Dialogs and popovers seem similar. How are they different?- Hidde Blog](https://hidde.blog/dialog-modal-popover-differences/)
Blogpost waarin Hidde de verschillen tussen Dialogs en pop-overs uiteen zet. Daarin komt ook [Non Modal Dialog](https://hidde.blog/dialog-modal-popover-differences/#:~:text=Non%20modal%20dialog%3A%20while%20this%20chat%20widget%20is%20open%2C%20I%20can%20still%20access%20the%20forms%20and%20content%20underneath.) aan bod.

**Comment by jeffreylauwers** [comment]
## Den Haag

- [Storybook](https://nl-design-system.github.io/denhaag/?path=/docs/css-page-elements-floating-contact--docs)
- [GitHub](https://github.com/nl-design-system/denhaag/tree/main/components/FloatingContact)

---

### Screenshots / Varianten

**Storybook**

Default
![Scherm­afbeelding 2024-12-23 om 15 22 18](https://github.com/user-attachments/assets/5f8813c0-8882-4adf-8085-fa638da1a6cf)

Expanded
![Scherm­afbeelding 2024-12-23 om 15 22 33](https://github.com/user-attachments/assets/b960f948-830b-4972-8587-dafe9b0cb7d4)

---

### Notities

- Non Modal Dialog wordt ingezet voor contactmogelijkheden.
- Call to action als button heeft een fixed postition rechtsonder.

**Comment by jeffreylauwers** [comment]
## Rotterdam

[rotterdam.nl](https://www.rotterdam.nl/)

---

### Screenshots / Varianten

**rotterdam.nl**

![Scherm­afbeelding 2024-12-23 om 15 26 59](https://github.com/user-attachments/assets/eab20a65-11e2-4fd1-a0d1-d383ebace964)

---

### Notities

- Niet als 'los' component gevonden. Maar Non Modal Dialog wordt ingezet voor een functionaliteit om de website te vertalen.
- Non Modal Dialog opent door te klikken op de button 'Translate'. Daarna sprint de focus naar de Select binnen de Non Modal Dialog.
- De Non Modal Dialog heeft rechtsboven een sluit-functie.

**Comment by Yolijn** [comment]
## ✨ Dit component is nu Help Wanted ✨

Help je mee hem door de Community stappen te krijgen?
[Dialog op het Community bord](https://github.com/orgs/nl-design-system/projects/29/views/1?filterQuery=%22Dialog%22+181)
